{{cookiecutter.project_slug}}/backend/app/app/crud/item.py

```diff
@@ -5,51 +5,31 @@
 
 from app.db_models.item import Item
 from app.models.item import ItemCreate, ItemUpdate
+from app.crud.base import CrudBase
 
 
-def get(db_session: Session, *, id: int) -> Optional[Item]:
-    return db_session.query(Item).filter(Item.id == id).first()
+class CrudItem(CrudBase):
+    """
+    This is provided as a showcase of which methods to override, with the benefit to adjusting
+    both the types of the arguments and of the returned objects to the proper 'Item*' classes
+    """
 
+    def get(self, db_session: Session, id: int) -> Optional[Item]:
+        return super(CrudItem, self).get(db_session, obj_id=id)
 
-def get_multi(db_session: Session, *, skip=0, limit=100) -> List[Optional[Item]]:
-    return db_session.query(Item).offset(skip).limit(limit).all()
+    def get_multi(self, db_session: Session, *, skip=0, limit=100) -> List[Optional[Item]]:
+        return super(CrudItem, self).get_multi(db_session, skip=skip, limit=limit)
 
+    def create(self, db_session: Session, *, item_in: ItemCreate, owner_id: int) -> Item:
+        item_in_data = jsonable_encoder(item_in)
+        item = Item(**item_in_data, owner_id=owner_id)
+        db_session.add(item)
+        db_session.commit()
+        db_session.refresh(item)
+        return item
 
-def get_multi_by_owner(
```

Comment:
I think we are now missing get_multi_by_owner, but it's still used in the api. So even though it doesn't show up in tests, it might be worth adding.

Reply:
you are entirely right. nice catch 👍

Reply:
Added back, making use of `get_multi_by` from `CrudBase`